regulations/tests/layers_footnotes_tests.py

```diff
@@ -0,0 +1,68 @@
+from unittest import TestCase
+
+from regulations.generator.layers.footnotes import FootnotesLayer
+
+
+class FootnotesLayerTest(TestCase):
+    def test_single_note(self):
+        layer = {
+            "555-220-p1": [
+                {
+                    "footnote_data": {
+                        "note": "Some notes",
+                        "ref": "1"
+                    },
+                    "locations": [
+                        0
+                    ],
+                    "text": "[^1]Some actual text"
+                }
+            ]
+        }
+        fl = FootnotesLayer(layer)
+        self.assertEqual(fl.apply_layer('555-220-p1'), ('footnotes',
```

Comment:
This isn't _quite_ the use case we're going for -- I think you want '555-220' rather than '555-220-p1'